runtime/notification.go

```diff
@@ -0,0 +1,71 @@
+package runtime
+
+import (
+	"fmt"
+
+	"github.com/mitchellh/mapstructure"
+)
+
+type Notification struct {
+	GroupID   string      `json:"group_id"`
```

Comment:
What does the group ID map to? We need godocs for all these fields and types ;-)

Reply:
`GroupID` is the group id for the test instance that emits that notification. We then collect all notifications for a given group and visualise them:

![Uploading Screenshot 2020-10-02 at 16.11.28.png…]()

Will add godoc.